.github/workflows/build-vsix.yaml

```diff
@@ -29,3 +34,17 @@ jobs:
         with:
           name: mltools-vscode
           path: skyline-vscode/skyline-vscode-*.vsix
+
+      - name: Upload vsix artifact to S3
+        run: |
+          aws s3 cp skyline-vscode/skyline-vscode-*.vsix s3://centml-habitat/skyline-vscode/
```

Comment:
are you missing the `-r` recursive flag here?

Reply:
We don't need the recursive flag as we only build one version of the artifact